app/models/manageiq/providers/cloud_automation_manager/configuration_manager/refresher.rb

```diff
@@ -41,13 +50,14 @@ def collect_configuration_inventory(connection, base_url)
         team = "default"
         # Get all templates
         all = "all"
-        # get all templates
-        template_uri_str = base_url + ":30000/cam/api/v1/templates?tenantId=" + tenant_id + "&ace_orgGuid=" + all + "&cloudOE_spaceGuid=" + team
-        # get stacks
-        #template_uri_str = base_url + ":30000/cam/api/v1/stacks?tenantId=" + tenant_id + "&ace_orgGuid=" + all + "&cloudOE_spaceGuid=" + team
+        # url to get all template resources
+        template_uri_str = base_url + api_url + "?tenantId=" + tenant_id + "&ace_orgGuid=" + all + "&cloudOE_spaceGuid=" + team
         template_body = redirect_cam_api(template_uri_str, 5, connection)
-        result[:templates] = JSON.parse(template_body.body)
-
+        if type == "templates"
```

Comment:
I would probably put this logic on the caller side and return `JSON.parse(template_body.body)`